xcodeproject/xcodeproj/plist.go

```diff
@@ -1,11 +1,10 @@
 package xcodeproj
 
 import (
-	"io/ioutil"
-
 	plist "github.com/bitrise-io/go-plist"
 	"github.com/bitrise-io/go-utils/fileutil"
 	"github.com/bitrise-io/go-xcode/xcodeproject/serialized"
+	"os"
```

Comment:
Could you move this stdlib import to the top and in a separate group?

Reply:
Fixed here: https://github.com/bitrise-io/go-xcode/pull/247/commits/2ae74667208cbbfa70df4696010105bfc3bb15ed